.neutrinorc.js

```diff
@@ -0,0 +1,107 @@
+const acceptedExternalEnvs = {
+  BACKEND: 'BACKEND' in process.env ?
```

Comment:
I've been trying to use Heroku Review apps and I get the feeling that if I change the env variable "post-build" it will take no effect.
Unfortunately Heroku Review apps do not allow to do a deploy from a branch as you can with normal apps, thus, needing to push code to cause another build which would include the env variable.

Restarting the Dynos of the Review App won't build again, thus, failing to use the new variable.

Pushing empty commits does not rebuild either.

The steps to make this work:
* set the env variable pointing to a different backend than the default one
* force push a dummy change

Is there a different way to rebuild a review app?